ocaml/idl/datamodel.ml

```diff
@@ -6577,6 +6577,17 @@ let vdi_disable_cbt = call
     ~allowed_roles:_R_VM_ADMIN
     ()
 
+let vdi_set_cbt_enabled = call
```

Comment:
I think it would be helpful to add a comment here and at the implementation saying that this is for internal use by SM to set the cbt_enabled field when it disables it on its own for some reason, and that it should probably be removed when SMAPIv3 arrives.

Reply:
Agreed.